plugins/game/services/UserScoreService.php

```diff
@@ -55,4 +55,19 @@ public function updateAction($score, KalturaUserScorePropertiesFilter $filter)
 		
 		return $filter->updateUserScore($score, $filter);
 	}
+	
+	/**
+	 * @action delete
+	 * @param KalturaUserScorePropertiesFilter $filter
+	 * @return KalturaUserScorePropertiesResponse
+	 */
+	public function deleteAction($filter)
+	{
+		if (!$filter)
+		{
```

Comment:
the action should get the object type, object id and user id.
filter is not the correct field
we should align it in update action as well